frontend/api/index.js

```diff
@@ -108,6 +108,7 @@ app.get('/config/project-overrides', (req, res) => {
     { name: 'amplitude', value: process.env.AMPLITUDE_API_KEY },
     { name: 'delighted', value: process.env.DELIGHTED_API_KEY },
     { name: 'capterraKey', value: process.env.CAPTERRA_API_KEY },
+    { name: 'stale_flags_limit_days', value: process.env.STALE_FLAG_LIMIT_DAYS || "30" },
```

Comment:
This is an attribute on the project, although I see that I haven't exposed it in the serializer. I will do that now, but you should update this PR to add a configurable attribute in the project settings.

Reply:
Gotcha, I missinterpreted what Project. stale_flags_limit_days, will remove